src/Cli.Tests/Snapshots/UpdateEntityTests.TestUpdateRestAndGraphQLSettingsForStoredProcedures_c9b12fe27be53878.verified.txt

```diff
@@ -40,6 +40,9 @@
           Operation: Mutation
         },
         Rest: {
+          Methods: [
```

Comment:
When `Enabled` property is set to `false`, the REST APIs are disabled for the entity (tested this manually). However, when writing to the config json, the `methods` and `path` fields are written. This is the same case for `graphql` field too. 

Clean-up of this can be taken up in a subsequent PR